extensions/amp-auto-lightbox/0.1/amp-auto-lightbox.js

```diff
@@ -438,13 +453,17 @@ export function runCandidates(ampdoc, candidates) {
     whenLoaded(candidate).then(() => {
       return measureIntersectionNoRoot(candidate).then(
         ({boundingClientRect}) => {
-          // <amp-img> will change the img's src inline data on unlayout and
-          // remove it from DOM.
-          if (!candidate.signals().get(CommonSignals.LOAD_END)) {
+          if (
+            !candidate.tagName === 'IMG' &&
```

Comment:
This looks like a bug
```suggestion
            candidate.tagName !== 'IMG' &&
```